src/components/table/table.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ChangeEvent, useState } from 'react';
+import { render } from '@testing-library/react';
+
+import { testA11y } from '../../test-utils';
+import { Table, TableCaption, Tbody, Td, Th, Thead, Tr } from './table';
+
+const tableHead = () => (
+  <Thead>
+    <Tr>
+      <Th>Month you apply</Th>
+      <Th>Rate for bicycles</Th>
+      <Th>Rate for vehicles</Th>
+    </Tr>
+  </Thead>
+);
+
+const tableBody = () => (
+  <Tbody>
+    <Tr>
+      <Td>January</Td>
+      <Td isNumeric>£85</Td>
+      <Td isNumeric>£95</Td>
+    </Tr>
+    <Tr>
+      <Td>February</Td>
+      <Td isNumeric>£75</Td>
+      <Td isNumeric>£55</Td>
+    </Tr>
+    <Tr>
+      <Td>March</Td>
+      <Td isNumeric>£165</Td>
+      <Td isNumeric>£125</Td>
+    </Tr>
+  </Tbody>
+);
+
+test('it displays a basic table passes a11y', async () => {
+  const { container } = render(<Table>{tableBody()}</Table>);
+  expect(container).toMatchSnapshot();
+  await testA11y(container);
+});
+
+test('it displays a custom styled basic table passes a11y', async () => {
+  const tableClass = 'customTableClass';
+  const { container } = render(
+    <Table className={tableClass}>{tableBody()}</Table>
+  );
+  expect(container).toMatchSnapshot();
+  await testA11y(container);
+});
+
+test('it displays a table with a header and passes a11y', async () => {
+  const { container } = render(
+    <Table>
+      {tableHead()}
```

Comment:
This is a strange syntax to render a react component. Cant you just do:

```tsx
const TableBodySample = () => (
  <Tbody>
    <Tr>
      <Td>January</Td>
      <Td isNumeric>£85</Td>
      <Td isNumeric>£95</Td>
    </Tr>
    <Tr>
      <Td>February</Td>
      <Td isNumeric>£75</Td>
      <Td isNumeric>£55</Td>
    </Tr>
    <Tr>
      <Td>March</Td>
      <Td isNumeric>£165</Td>
      <Td isNumeric>£125</Td>
    </Tr>
  </Tbody>
);

....

<TableBodySample />
```

Feel free to change up the naming